### PR TITLE
Release workflow

### PR DESCRIPTION
**Pull request by kratman**
Adding a release workflow

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,17 +23,14 @@ dev = [
...
     "myst-parser",
 ]
 
-[tools.setuptools]
-packages = [
-    { include = "pbeis" },
-]
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["pbeis"]
```
Comment (rtimms) [inline pyproject.toml:33]: should we also rename this to pybammeis? so it's not confusing for users? same for examples etc.?

also, the installation instructions can be updated for a user install

**Review by rtimms — APPROVED** [review]
thanks!

File: pyproject.toml (hunk 1 above)
Comment (kratman) [reply pyproject.toml:33]: I can rename that as well if you want, but having different names is possible. The way I have it set up now is that you install with
```
pip install pybammeis
```
but still import with
```
import pbeis
```

If you want them to be same that is fine though.

Comment (rtimms) [reply pyproject.toml:33]: yeah I think it might be confusing if they are different

**Review by rtimms — APPROVED** [review]

**Pull request merged**